src/application-delegate.coffee

```diff
@@ -233,6 +233,14 @@ class ApplicationDelegate
     new Disposable ->
       ipcRenderer.removeListener('context-command', outerCallback)
 
+  onUrlMessage: (callback) ->
```

Comment:
Stupid nitpick... we typically do initialisms like URL and HTML in all capital letters unless they're the first word of the identifier, so this should be `onURLMessage`. We do this to blend in with the majority of DOM APIs.